testproject/Assets/Tests/Runtime/MultiprocessRuntime/TestCoordinator.cs

```diff
@@ -51,7 +52,36 @@ private void Awake()
 
     public void Start()
     {
+        string connectAddress = "127.0.0.1";
         bool isClient = Environment.GetCommandLineArgs().Any(value => value == MultiprocessOrchestration.IsWorkerArg);
+        string[] args = Environment.GetCommandLineArgs();
+        foreach (string arg in args)
+        {
+            if (arg.StartsWith("-ip="))
+            {
+                connectAddress = arg.Replace("-ip=", "");
+                Debug.Log($"connectAddress is {connectAddress}");
+            }
+        }
+
+        string port = "3076";
```

Comment:
Same question here about the string to ushort.  Is this going to be a command line parameter?  If so, then why not add it above when it is parsing the arguments?
i.e.: arg.StartsWith("-port=")

Just curious.  :)

Reply:
I did intend to make this a parameter eventually ... I'll make this change.